cluster/job_once_example_test.go

```diff
@@ -1,13 +1,15 @@
 package cluster
 
 import (
+	"log"
 	"time"
 
 	"github.com/mattermost/mattermost-server/v6/plugin"
 )
 
-func HandleJobOnceCalls(key string) {
+func HandleJobOnceCalls(key string, props any) {
 	if key == "the key i'm watching for" {
+		log.Println(props)
```

Comment:
Do we want this print?

Reply:
Yes, I found that useful as part of the example.